osu.Game/Graphics/OsuColour.cs

```diff
@@ -156,13 +156,13 @@ public Color4 ForModType(ModType modType)
                     return Lime1;
 
                 case ModType.Conversion:
-                    return Purple1;
+                    return Purple0;
 
                 case ModType.Fun:
-                    return Pink1;
+                    return Pink0;
 
                 case ModType.System:
-                    return Gray7;
+                    return Gray5;
```

Comment:
I recall deviating from design for a reason here too but I cannot recall it now. This one is probably fine to keep.